data/personas/it_manager_analytical.yaml

```
@@ -0,0 +1,102 @@
+
+vocabulary_level: professional
+response_speed: thoughtful
+humor_level: 0.3
+
+# シナリオ別の振る舞い設定例
+scenario_behaviors:
+  deadline_pressure:
+    stress_level: high
+    cooperation: focused
+    communication: direct
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)